docs/source/best-practices.md

```diff
@@ -186,4 +186,5 @@ You should not use [Spark caching](optimizations/delta-cache.md#delta-and-rdd-ca
 
 - The data that gets cached may not be updated if the table is accessed using a different identifier (for example, you do `spark.table(x).cache()` but then write to the table using `spark.write.save(/some/path)`.
 
-.. include:: /shared/replacements.md
+.. <Delta> replace:: Delta Lake
```

Comment:
why replace this?

Reply:
seems like a lot of duplication now.

Reply:
Don't know the reason, but having the `./shared/replacement.md` in every file cause the side navigation to break.

Reply:
WOW.